src/main/java/duke/Event.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    protected LocalDate at;
+
+    public Event(String desc, String at) {
+        super(desc);
+        this.at = LocalDate.parse(at);
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + at.format(DateTimeFormatter.ofPattern("MMM d yyyy")) + ")";
+    }
+
+    @Override
+    public String toDisk() {
+        return String.format("event\n%s\n%d\n%s", desc, (done ? 1 : 0), at);
+    }
+}
```

Comment:
Should there be a trailing whitespace here? Also, perhaps it might be good to add a new line to end of file.